src/index.js

```diff
@@ -19,4 +21,9 @@ const config = {
 
 firebase.initializeApp(config);
 
-ReactDOM.render(<Root />, document.getElementById('root'));
+ReactDOM.render(
+  <I18nextProvider i18n={i18next}>
```

Comment:
It would be better to move this wrapper inside the root component to keep the index simple.

Reply:
done